core/src/main/java/bio/terra/pearl/core/service/publishing/PublishingSupport.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.pearl.core.service.publishing;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import bio.terra.pearl.core.model.Versioned;
+import bio.terra.pearl.core.model.portal.PortalEnvironment;
+import bio.terra.pearl.core.model.publishing.*;
+import bio.terra.pearl.core.model.study.StudyEnvironment;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public interface PublishingSupport {
```

Comment:
maybe simply `Publishable`? I feel like interfaces tell you what a thing is `able` to do.